gulpfile.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+var gulp = require('gulp');
+var jshint = require('gulp-jshint');
+
+gulp.task('test', ['test:jshint']);
+
+gulp.task('test:jshint', function() {
+    return gulp.src(['**/*.js', '!staticfiles/**/*.js', '!node_modules/**/*.js', '!env/**/*.js'])
+               .pipe(jshint())
+               .pipe(jshint.reporter('default'));
+});
```

Comment:
Unless you're planning on adding more JS, it's probably easier to make a more specific glob pattern here since we only have 3 *.js files currently:

```
$ find . | grep -v "node_modules" | grep "\.js$"

./dashboard/static/js/analytics.js
./dashboard/static/js/sign_in.js
./gulpfile.js
```

---

```js
gulp.task('test:jshint', function() {
    return gulp.src(['*.js', './dashboard/static/js/**/*.js'])
               .pipe(jshint())
               .pipe(jshint.reporter('default'));
});
```